crates/kitsune_p2p/types/benches/api_thru.rs

```diff
@@ -1,148 +1,3 @@
-
-//         let con = self
-//             .src_ep
-//             .get_connection(self.dst_url.as_str(), t)
-//             .await
-//             .unwrap();
-
-//         let res = con
-//             .request(&data, KitsuneTimeout::from_millis(5000))
-//             .await
-//             .unwrap();
-
-//         if let TestData::One(One { data }) = res {
-//             assert_eq!(data.as_ref(), RES);
-//         } else {
-//             panic!("invalid data");
-//         }
-//     }
-// }
-
-// async fn mk_core() -> (TxUrl, Tx2Ep<TestData>, Tx2EpHnd<TestData>) {
-//     let t = KitsuneTimeout::from_millis(5000);
-
-//     let f = tx2_mem_adapter(MemConfig::default()).await.unwrap();
-//     let f = tx2_pool_promote(f, Default::default());
-//     let f = tx2_api(f, Default::default());
-
-//     let ep = f.bind(TxUrl::from_str_panicking("none:"), t).await.unwrap();
-//     let ep_hnd = ep.handle().clone();
-//     let addr = ep_hnd.local_addr().unwrap();
-
-//     (addr, ep, ep_hnd)
-// }
-
-// async fn mk_dst() -> (TxUrl, Tx2EpHnd<TestData>) {
-//     let (url, mut ep, ep_hnd) = mk_core().await;
-
-//     tokio::task::spawn(async move {
-//         while let Some(evt) = ep.next().await {
-//             if let Tx2EpEvent::IncomingRequest(Tx2EpIncomingRequest { data, respond, .. }) = evt {
-//                 if let TestData::One(One { data }) = data {
-//                     assert_eq!(data.as_ref(), REQ);
-//                 } else {
-//                     panic!("invalid data");
-//                 }
-//                 let mut data = PoolBuf::new();
-//                 data.extend_from_slice(RES);
-//                 let data = TestData::one(data);
-//                 respond
-//                     .respond(data, KitsuneTimeout::from_millis(5000))
-//                     .await
-//                     .unwrap();
-//             }
-//         }
-//     });
-
-//     (url, ep_hnd)
-// }
-
-// async fn mk_src() -> Tx2EpHnd<TestData> {
-//     let (_url, mut ep, ep_hnd) = mk_core().await;
-
-//     tokio::task::spawn(async move { while let Some(_evt) = ep.next().await {} });
-
-//     ep_hnd
-// }
-
-// async fn test(this: &Share<Option<Test>>) {
-//     let mut t = this.share_mut(|i, _| Ok(i.take().unwrap())).unwrap();
-//     t.test().await;
-//     this.share_mut(move |i, _| {
-//         *i = Some(t);
-//         Ok(())
-//     })
-//     .unwrap();
-// }
-
-// fn api_thru(rt: &tokio::runtime::Runtime, t: &Share<Option<Test>>) {
-//     rt.block_on(async {
-//         test(t).await;
-//     });
-// }
-
-// fn criterion_benchmark(c: &mut Criterion) {
-//     let rt = tokio::runtime::Builder::new_multi_thread()
-//         .enable_all()
-//         .build()
-//         .unwrap();
-
-//     let t = rt.block_on(async { Share::new(Some(Test::new().await)) });
-
-//     c.bench_function("api-thru-mem", |b| b.iter(|| api_thru(&rt, &t)));
-// }
-
-// criterion_group!(benches, criterion_benchmark);
-// criterion_main!(benches);
```

Comment:
@neonphog useful?

Reply:
I'd say this whole file should be deleted. If we're writing tx5 benchmarks they should be in the tx5 repo.